FlowCrypt/Functionality/Services/WKDURLsApi.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  WKDURLsService.swift
+//  FlowCrypt
+//
+//  Created by Yevhen Kyivskyi on 23.05.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import CryptoKit
+import Promises
+
+protocol WKDURLsApiType {
+    func lookupEmail(_ email: String) -> Promise<[String]>
+    func rawLookupEmail(_ email: String) -> Promise<CoreRes.ParseKeys?>
+}
+
+class WKDURLsApi: WKDURLsApiType {
+
+    private let wkdURLsConstructor: WKDURLsConstructorType
+    private let core: Core
+
+    init(
+        wkdURLsConstructor: WKDURLsConstructorType = WKDURLsConstructor(),
+        core: Core = Core.shared
+    ) {
+        self.wkdURLsConstructor = wkdURLsConstructor
+        self.core = core
+    }
+
+    func lookupEmail(_ email: String) -> Promise<[String]> {
```

Comment:
Here we are getting the raw bytes, then parsing them, and then armoring them and returning armored result.

Somewhere down the line another code will de-armor, parse the result to actually use the keys.

Best would be to change the signature of returns of both AttesterApi and this class have a public signature only `lookupEmail(_ email: String) -> Promise<[KeyDetails]>` - just one method